indra/databases/obo_client.py

```diff
@@ -70,6 +60,122 @@ def __init__(self, prefix, *, directory=RESOURCES):
         self.synonym_to_id = {k: v for k, v in self.synonym_to_id.items()
                               if k not in ambig_synonyms}
 
+    def count_xrefs(self):
```

Comment:
@bgyori this is not properly implemented and never used as far as I can tell. Can I remove it?

Reply:
*except in the `__main__`  of some derived clients. Maybe the implementation never quite made it to master

Reply:
It looks like that's something you added originally, I don't think we have any usages so feel free to remove.

Reply:
lol

Reply:
Addressed in 21e431363